public/data/platforms.json

```diff
@@ -104,6 +104,19 @@
     },
 
     {
+	"name": "linux-x86_64-centos6-epel-rdt",
+	"description": "CentOS 6 with Redhat Developer Toolset, R from EPEL",
+	"cran-name": null,
+	"rversion": "r-release",
+	"os-type": "Linux",
+	"cpu-type": "x86_64",
+	"os-info": "CentOS 6",
+	"compilers": "GCC 5.2.1",
+	"docker-image": "centos6-epel",
+	"sysreqs-platform": "linux-x86_64-centos6-epel-rdt"
```

Comment:
Or at least all entries here: https://github.com/r-hub/sysreqsdb/blob/master/platforms/linux-x86_64-centos6-epel.json apply to this platform, right?